data/categories.txt

```diff
@@ -3476,7 +3477,7 @@ ja:クリニック|医師|医者|ドクター|救急|診療
 pl:klinika|lekarz|doktor|leczenie
 pt:3Hospital|2Clínica|3médico|serviços de saúde|primeiros socorros
 pt-BR:Hospital|3clínica|3médico|saúde|serviços de saúde|primeiros socorros
-ro:Centru medical|ambulanță|medic|doctor|diagnostică
+ro:Spital|ambulanță|medic|doctor|diagnostică
```

Comment:
Here "spital" is redundant also, because its defined in `@hospital` already (at line 3450).